srcs/layouts/shaw_imperial_en.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Shaw Imperial layout for shavian alphabet, see https://www.shavian.info/keyboards/ -->
+<keyboard name="Shaw Imperial" script="shavian">
+  <row>
```

Comment:
4 rows is a lot and corners are underutilized.
Any way of removing one by moving some characters to the corners of other keys ?
The "Shaw QWERTY" layout manages to have only 3 rows.

Reply:
Yes, I agree, that four rows make it rather tall, but I don't know if it is okay to move a whole row to corners of adjacent row. Shavian just has more letters than Latin. I think this change would make it a different layout... I'll test to see if I can come up with something that works well.

Reply:
Matching the official layout is also nice! That was just a suggestion, 4 rows is fine if it's the layout that you would use. Same for the alternating shift, choose what you'd prefer using.

Reply:
Well, I came up with something that works okay. I'm thinking this: will it be okay if I add both versions? So it would be like "Shaw Imperial" and "Shaw Imperial (Compact)".

I could add Shaw QWERTY too, btw.

Reply:
The layout selector is not ergonomic enough to handle too many layouts so I don't accept small variations and layouts without users.
But feel free to post any number of layout to https://github.com/Julow/Unexpected-Keyboard-layouts, which have a chance to be included in the app when the settings improve.